Where_Is_My_Wife/Assets/_Main/LevelMusicChanger.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using WhereIsMyWife.Managers;
+public class LevelMusicChanger : MonoBehaviour
+{
+    void Start()
+    {
+        AudioManager.Instance.PlayMusic(SceneManager.GetActiveScene().name,true);
+    }
+    public void ChangeSceneWithMusic(string sceneName)
+    {
+        AudioManager.Instance.ChangeSceneWithFadeOut(sceneName);
+    }
+}
```

Comment:
This class is very clumped, is not part of any namespace and its methods have no empty line in between.